Client/Components/DashBoard.razor

```diff
@@ -3,12 +3,15 @@
 
 
 <div class="dash-board">
-    @if (dataMetrics.Count() == 0)
+    @if (countryComp == null || countryCompTwo == null)
```

Comment:
Could these maybe have clearer names?

Reply:
I used the names that already existed. I am not sure if we are supposed to change names that have been previously defined?